src/util.h

```diff
@@ -191,6 +191,16 @@ bool OpenDebugLog();
 void ShrinkDebugFile();
 void runCommand(const std::string& strCommand);
 
+/**
+ * Most paths passed as configuration arguments are treated as relative to
+ * the datadir if they are not absolute.
+ *
+ * @param path The path to be conditionally prefixed with datadir.
+ * @param net_specific Forwarded to GetDataDir().
+ * @return The normalized path.
+ */
+fs::path GetAbsolutePathForArg(const fs::path& path, bool net_specific = true);
```

Comment:
"ForArg" in function name seems a little redundant. Just a suggestion, but I might call it something `GetDataPath` or `GetDataDirPath`.

Reply:
My only reservation here is that the path won't necessarily be in datadir, so a name with Data or DataDir seems confusing. "Arg" here is meant to correspond to an argument to the binary, not the function, but maybe that isn't clear.

Reply:
I'll try `AbsPathForConfigVal`.